N40: godot-rust (#1217)
rust-gamedev/rust-gamedev.github.io: content/news/040/index.md

[SpriteRef]: https://docs.rs/pixel_engine/0.6.0/pixel_engine/graphics/struct.SpriteMutRef.html
[wgpu]: https://wgpu.rs/

### [godot-rust][gd-github]

![godot-rust GDExtension](godot-rust-gdextension.png)

godot-rust ([GitHub][gd-github], [Discord][gd-discord], [Twitter][gd-twitter])

> is a Rust library that provides bindings for the Godot engine. Just this month,
> we also opened a [Mastodon account][gd-mastodon] to share development info.

Reviewer: ```suggestion
is a Rust library that provides bindings for the Godot engine. Just this month, a [Mastodon account][gd-mastodon] was opened to share development info.
```
